fair_mango/metrics/metrics.py

```diff
@@ -82,6 +249,40 @@ class ConfusionMatrix(Metric):
     - false negative rate
```

Comment:
stay coherent in your descriptions. If for selectionrate you specify "for different sensitive groups" then do it here and for perfomance metric too.

Reply:
Done!